source/_data/SymbioticLab.bib

```diff
@@ -654,7 +654,7 @@ @article{hsct-survey
   year={2020},
   publisher={Multidisciplinary Digital Publishing Institute},
 
-  publist_confkey = {Sensors 20(21)},
+  publist_confkey = {Sensors:20(21)},
```

Comment:
I think you need to change the "confkey" in https://github.com/SymbioticLab/SymbioticLab.github.io/blob/develop/source/publications/index.md